File: apps/anyupp-mobile/lib/shared/models/group.dart
import 'address.dart';
import 'payment.dart';

class Group {
  String chainId;
  String name;
  String email;
  String phone;
  String profileImage;
  Address address;
  Payment payment;
  String description;
  Map<String, String> products;

  Group({
    this.chainId,
    this.name,
    this.email,
    this.phone,
    this.profileImage,
    this.address,
    this.payment,
    this.description,
    this.products,
  });
}

File: libs/shared/types/src/lib/interfaces/group.ts
import { IAddressInfo } from './address';
import { IContact } from './contact';
import { ILocalizedItem } from './localized-item';

export interface IGroup extends IContact, IAddressInfo {
  __typename?: 'Group';
  id: string;
  chainId: string;
  name: string;
  description: ILocalizedItem<string>;
  currency: string; // group edit, unit readonly
  createdAt: string;
  updatedAt: string;
}
